### PR TITLE
New package: Egnyte.EgnyteWebEdit version 2.4.900.67

### LINKED ISSUE #154091: Egnyte Webedit package request
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://egnyte-cdn.egnyte.com/webedit/win/en-us/latest/EgnyteWebEdit.msi

Publisher: Egnyte
Package Name: Egnyte Webedit
Description: 
Package Version: 2.4.9
Installer URL: https://egnyte-cdn.egnyte.com/webedit/win/en-us/latest/EgnyteWebEdit.msi

### PR DESCRIPTION
**Pull request by damn-good-b0t**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.2.1 :rocket:

resolves #154091

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/154147)

### CODE REVIEW

**Review by Trenly — APPROVED** [review]

**Comment by stephengillie** [comment]
The `\hc` in the path suggests that Egnyte are probably customers of ZenDesk, and using their Help Center software. Unfortunately, ZenDesk have blocked our scans.

File: manifests/e/Egnyte/EgnyteWebEdit/2.4.900.67/Egnyte.EgnyteWebEdit.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Egnyte.EgnyteWebEdit
+PackageVersion: 2.4.900.67
+PackageLocale: en-US
+Publisher: Egnyte, Inc
+PublisherUrl: https://www.egnyte.com/
+PackageName: Egnyte WebEdit
+PackageUrl: https://helpdesk.egnyte.com/hc/en-us/articles/201637704-Egnyte-WebEdit
```
Comment (stephengillie) [inline manifests/e/Egnyte/EgnyteWebEdit/2.4.900.67/Egnyte.EgnyteWebEdit.locale.en-US.yaml:10]:
```suggestion
# PackageUrl: 
```

File: manifests/e/Egnyte/EgnyteWebEdit/2.4.900.67/Egnyte.EgnyteWebEdit.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+PackageLocale: en-US
+Publisher: Egnyte, Inc
+PublisherUrl: https://www.egnyte.com/
+PackageName: Egnyte WebEdit
+PackageUrl: https://helpdesk.egnyte.com/hc/en-us/articles/201637704-Egnyte-WebEdit
+License: Proprietary
+Copyright: Copyright (c) Egnyte, Inc
+ShortDescription: Egnyte WebEdit enables you to select a file from our Web UI and automatically open the file in its native application.
+Description: Egnyte WebEdit enables you to select a file from our Web UI and automatically open the file in its native application. Changes you save will automatically appear in the cloud as new versions. This lightweight product is ideal for users who need to edit files on their desktops, but don’t want to install Desktop App or Desktop Sync. It's also more convenient than downloading, editing, saving, and uploading the saved version back to the cloud.
+ReleaseNotesUrl: https://helpdesk.egnyte.com/hc/en-us/articles/17360622113549-Minor-Improvements-in-WebEdit-2-4-9-for-Windows-and-Mac
```
Comment (stephengillie) [inline manifests/e/Egnyte/EgnyteWebEdit/2.4.900.67/Egnyte.EgnyteWebEdit.locale.en-US.yaml:15]:
```suggestion
# ReleaseNotesUrl: 
```

**Review by stephengillie — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files\Egnyte\Egnyte WebEdit\client.exe returned exit code: 70

(Automated response - build 881.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files\Egnyte\Egnyte WebEdit\client.exe returned exit code: 70

(Automated response - build 881.)

**Comment by stephengillie** [comment]
I'm not sure what `exit code: 70` indicates, but the install appears to be successful.

**Comment by stephengillie** [comment]
Verify: 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**